added keywords and errors for RM6174 Multi Functional Devices

dorota-hawro opened the pull request.

dorota-hawro committed:
added keywords and errors for RM6174 Multi Functional Devices

dorota-hawro committed:
changed domain name to Print and Digital, updated the keyword queries, modified error message for MFD

Chee-Ng commented on guided-match/dml-versions/v021.sql:
```diff
@@ -0,0 +1,108 @@
+
+-- CREATING THE JOURNEY
+INSERT INTO journeys (journey_id,journey_name,published)
+	VALUES ('af949053-6110-48a2-a252-ebb83fc98b14'::uuid,'Multi Functional Devices',true);
```
Name of the journey should be "Print and Digital"

Chee-Ng commented on guided-match/dml-versions/v021.sql:
```diff
@@ -0,0 +1,108 @@
+INSERT INTO temp_journeys values ('Print and Digital','af949053-6110-48a2-a252-ebb83fc98b14');
+
+insert into temp_domain_values values
+('Advice','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
```
remove the full stop at the end of the hint text for all keywords

Chee-Ng commented on guided-match/dml-versions/v021.sql:
```diff
@@ -0,0 +1,108 @@
+('Copier','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Copying','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Cost control opportunities','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+
```
remove whitespave

Chee-Ng commented on guided-match/dml-versions/v021.sql:
```diff
@@ -0,0 +1,108 @@
+('Technology','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Technology products','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.');
+
+
```
Ordering is incorrect after this point. You should be 

1. adding **temp_domain_values** keywords to **search_terms**
2. map new keywords with GM Lite
3. add existing keyword to **temp_domain_values**
4. then you can add everything from **temp_domain_values** to **search_domains**

Take a look at v017.sql and follow the structure

dorota-hawro replied:
Makes sense, thanks for the feedback Chee!

dorota-hawro committed:
query cleanup:
- removed full stop and whitespace
- changed version description and journey name to Print and Digital
- corrected query ordering
